server/controllers/events/testfixtures/test-repos/automerge/exp-output-autoplan.txt

```diff
@@ -4,10 +4,7 @@ Ran Plan for 2 projects:
 1. dir: `dir2` workspace: `default`
 
 ### 1. dir: `dir1` workspace: `default`
-<details><summary>Show Output</summary>
-
```

Comment:
actually, this should not be removed?

Reply:
This should be removed. Same with https://github.com/runatlantis/atlantis/pull/2947#discussion_r1063983266.

Terraform logs are trimmed spaces, so output log contains less than 12lines. So become unwrapped.